core/sql/regress/hive/EXPECTED015

```diff
@@ -1571,6 +1582,7 @@ Task:  CLEANUP         Status: Ended      Time: 2016-12-11 01:58:54.934
 Task:  CLEANUP         Status: Ended      Elapsed Time:    00:00:00.682
 Task:  LOADING DATA    Status: Started    Time: 2016-12-11 01:58:54.934
        Rows Processed: 5000 
+       Error Rows:     0
```

Comment:
A few tests with error rows will be nice.  Some of the current tests with 0 errors could be altered to have error rows, that way total time taken to run the test will not increase much.